Makefile

```diff
@@ -16,6 +16,8 @@ help: # Show how to get started & what targets are available
 	@echo
 
 GIT_REVISION := $(shell git rev-parse --short HEAD)
+GO_PACKAGES := `go list -f {{.Dir}} ./... | grep -v /vendor/`
```

Comment:
nit: replace `command` with $(command)

Reply:
@marcosnils: just updated, please help to review, thx